docs/output/representatives.md

```diff
@@ -311,9 +311,9 @@ Representative phone
   | name | [RepresentativeName](#representativename)| Name for an individual |
   | phone | [RepresentativePhone](#representativephone)| Phone for an individual |
   | email | `string`| Email Address.  string <email> <= 255 characters |
-  | address | [RepresentativeAddress](#representativeaddress)| Address for an individual. |
-  | birthDate | [RepresentativeBirthDate](#representativebirthdate)| Birthdate for an individual |
-  | governmentID | [RepresentativeGovernmentID](#representativegovernmentid)|  |
+  | address | [RepresentativeAddress](#representativeaddress)| Address for an individual |
+  | birthDate | [RepresentativeBirthDate](#representativebirthdate)| Birth date for an individual |
+  | governmentID | [RepresentativeGovernmentID](#representativegovernmentid)| SSN, or ITIN |
```

Comment:
I know we provide it elsewhere but maybe it's helpful to have the acronyms written out?